ds_judgements_public_ui/templates/layouts/base.html

```diff
@@ -34,7 +34,11 @@
     {% include "includes/cookie_consent/cookie_banner.html" %}
     <a id="skip-to-main-content" href="#main-content">{% translate "skiplink" %}</a>
     {% include "includes/environment_banner.html" %}
-    {% include "includes/phase_banner.html" %}
+    {% flag v2_homepage %}
+      { % include "includes/beta_phase_banner.html" %}
```

Comment:
With that space on my machine the beta phase banner doesn't render
```suggestion
      {% include "includes/beta_phase_banner.html" %}
```

Reply:
@jlhdxw thanks accidental space removed